### PR TITLE
INT-6886 new build, push, and certify for iq operator

### PR DESCRIPTION
**Pull request by jflinchbaugh**
build, push, and certify the iq operator with red hat.

JIRA: https://issues.sonatype.org/browse/INT-6886
Build: https://jenkins.ci.sonatype.dev/job/integrations/job/cloud/job/Red%20Hat/job/operator-iq-release/

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -4,49 +4,45 @@
...
-          string(credentialsId: 'rh-build-service-api-key', variable: 'API_KEY')]) {
-        runGroovy('ci/TriggerRedHatBuild.groovy', "'${version}' '${PROJECT_ID}' '${API_KEY}'")
+        string(
+          credentialsId: 'red-hat-scan-registry-password-nexus-iq-operator',
+          variable: 'REGISTRY_PASSWORD'),
+        string(
+          credentialsId: 'red-hat-api-token',
+          variable: 'API_TOKEN'),
+      ]) {
+        sh 'PATH="$PATH:." VERSION=$version ./build_red_hat_image.sh'
```
Comment (hectorhuol) [inline Jenkinsfile:40]: The same as in the other PR, don't you need to pass the `REGISTRY_PASSWORD` too as a parameter?

**Review by hectorhuol — APPROVED** [review]
+1 with a small question about the `REGISTRY_PASSWORD `

File: Jenkinsfile (hunk 1 above)
Comment (jflinchbaugh) [reply Jenkinsfile:40]: as mentioned in the other PR, `withCredentials` is importing it as REGISTRY_PASSWORD environment variable, which matches the name the script wants. (little `version` environment variable that the parameter added didn't match big `VERSION`, so I needed to explicitly set it here.)

**Review by eduard-tita — APPROVED** [review]
+1

**Pull request merged**